samples/MQL4/MovingAverages.mqh

```diff
@@ -0,0 +1,240 @@
+//+------------------------------------------------------------------+
+//|                                               MovingAverages.mqh |
+//|                   Copyright 2009-2013, MetaQuotes Software Corp. |

```

Comment:
Are these samples released under a license that permits redistribution?

From a glance, it doesn't look like it...